.eslintrc.cjs

```diff
@@ -3,7 +3,10 @@ module.exports = {
     node: true,
     es2021: true,
   },
```

Comment:
Of less importance, there is this `env` section and the `parserOptions` section below. These are configuring eslint to be aware of what environment and how to parse our code. 

The [`env`](https://eslint.org/docs/user-guide/configuring/language-options#specifying-environments) decides which variables are global, just pick all the environments you are working with - `node` (obviously) and `es2021` (the version of Javascript we are using).